sc3-prepare.R

```diff
@@ -2,75 +2,114 @@
 # Script to creare SCE object for clustering with SC3.
 # Carlos Talavera-López - 2018
 
-# Load optparse we need to check inputs
-
-suppressPackageStartupMessages(require(optparse))
-
-# Load common functions
-
-suppressPackageStartupMessages(require(workflowscriptscommon))
+# Load required packages
+packages <- c("tidyverse", "SingleCellExperiment", "scater", "optparse", "SC3","workflowscriptscommon")
+suppressPackageStartupMessages(lapply(packages, library, character.only = TRUE))
 
 # parse options
 
 option_list = list(
   make_option(
-    c("-c", "--counts"),
+    c("-o", "--SCE"),
```

Comment:
Could you keep argument naming for objects consistent with previous packages? e.g. we've been using  input-object-file

Reply:
See e.g. https://github.com/ebi-gene-expression-group/bioconductor-scater-scripts/blob/master/scater-extract-qc-metric.R